src/components/speaker/SpeakerListPromoted.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import Container from 'react-bootstrap/esm/Container'
+import SpeakerItem from './SpeakerItem'
+
+const SpeakerList = ({ speakers, title }) => (
```

Comment:
For consistency (and to match the filename) this function should be named `SpeakerListPromoted`.